app/Http/Controllers/Web/DevicesController.php

```diff
@@ -75,6 +83,12 @@ public function update(Request $request, Uuid $publicDeviceId): RedirectResponse
         }
 
         $properties = $request->all();
+        $currentUserId = $request->user()->id;
+
+        $validator = $this->validateDuplicateName($properties, $currentUserId);
```

Comment:
This chunk of code appears to be identical to the one above when creating a device except for the verb in the flash message.  Please consider making a private method with some parameter to prevent repeating this block of code.